src/app/domain/entities/manga/manga-entity.spec.ts

```diff
@@ -2,6 +2,20 @@ import { MangaEntity } from './manga-entity';
 
 describe('Mangá', () => {
   it('should create instance mangá entity', () => {
-    expect(new MangaEntity()).toBeTruthy();
+    const manga = new MangaEntity({
+      name: 'one piece',
+      total_pages: 1000,
```

Comment:
Maybe you could create a mock for this object.
You should have test to check if `manga.name` and `manga.total_pages` returns what you expected them to return. Your tests doesn't garantee that your manga have the atribbutes that you passed.